### PR TITLE
Repo docs - readme, contributing & code of conduct 

### PR DESCRIPTION
**Pull request by nicolebid**
The CONTRIBUTING.md and  CODE_OF_CONDUCT.md is complete. README.md needs some minor updates, but it's complete up to the point we're at now.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,22 @@
...
+
+With escalating inflation and growing environmental consciousness, consumers across Canada seek to save money where they can. Solar Savers, a user-friendly app, is designed to better equip homeowners in the decision-making process of purchasing solar panels for their home. With user's location, electricity rates, and panel quality, Solar Savers allows users to see the potential electricity and financial savings, helping them decide whether investing in solar panels is worthwhile.
+
+## Contributing
+
+Interested in contributing to this project? Check out the [contributing guidelines](CONTRIBUTING.md). *Please note: This project is released with a Code of Conduct. By contributing to this project, you agree to abide by the terms in the [Code of Conduct](CODE_OF_CONDUCT.md).* 
+
+## License
+
+Solar Savers is developed and maintained by Marco Bravo, Nicole Bidwell, Rafe Chang, and Sam Fo. The code is licensed under the MIT license. Please see `LICENSE.md` for details.
```
Comment (Marcony1) [inline README.md:15]: Overall, LGTM. However, should we also mention the Open Canada and Creative Commons Licenses here? (Although the pull request containing such change has not been approved yet).

**Review by Marcony1 — CHANGES_REQUESTED** [review]

**Review by rafecchang — APPROVED** [review]
looks good!

File: README.md (hunk 1 above)
Comment (Marcony1) [reply README.md:15]:
```suggestion
Solar Savers is developed and maintained by Marco Bravo, Nicole Bidwell, Rafe Chang, and Sam Fo. The code is licensed under the MIT License, Open Canada License, and Creative Commons License. Please see `LICENSE.md` for details.  
```

**Review by rafecchang — APPROVED** [review]
looks good!

**Review by rafecchang — APPROVED** [review]

**Review by Marcony1 — APPROVED** [review]

**Pull request merged**